src/buildstream/_project.py

```diff
@@ -396,6 +416,15 @@ def get_alias_uris(self, alias, *, first_pass=False, tracking=False):
         if not alias or alias not in config._aliases:  # pylint: disable=unsupported-membership-test
             return [None]
 
+        if self.junction:
+            parent_project = self.junction._get_project()
+            parent_alias = self.junction.get_parent_alias(alias)
+            if parent_alias:
+                # Delegate translation to parent project
+                return parent_project.get_alias_uris(parent_alias, first_pass=first_pass, tracking=tracking)
+            elif parent_project.disallow_subproject_uris:
```

Comment:
Same comment, is this logic still correct after changing `allow_subproject_uris` for `disallow_subproject_uris` ?